### DISCUSSION TITLE
Income & expense managementand mikrotik log.

### DISCUSSION THREAD

**Discussion by mdarparvez** [hotspotbilling/phpnuxbill#53] [Ideas]
It would be great if there's an income & expense management system, income report expense report. And also the ability to store and download Mikrotik IP log.

**Comment by ibnux** [comment]
Expenses I don't plan it. 


 Log maybe, i can create plugin for that

**Comment by netlinktech** [comment]
Hi, there i was already make expense controller and template in my case

**Reply by FiberhomeKe** [reply]
can you share please gideonmonari@gmail.com

**Comment by netlinktech** [comment]
need log report plugin in report section and also need csv,excel,pdf download option in customer section. i'm already tryto do this but filed
